server/routes/addValidator.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response, NextFunction } from 'express';
```

Comment:
This file is not necessary. All validator creation should happen in event handler not in a route.

Reply:
Wanted to add some data in this table thats why created the route.Will remove done in  get event-handlers

Reply:
Sounds good.